cmd/eskimo/users_test.go

```diff
@@ -0,0 +1,267 @@
+	var status int
+
+	SETUP("we cleanup existing users, if any", func() { bridge.MustDeleteAllUsers(ctx, t) })
+	GIVEN("empty DB", func() {})
+	WHEN("get all users", func() {
+		body, status = bridge.GetUsers(ctx, t, "u", token)
+	})
+	THEN(func() {
+		IT("is successfully returned", func() {
+			assert.Equal(t, 200, status)
+		})
+		IT("should be empty", func() {
+			assert.Equal(t, body, "[]")
+		})
+	})
+}
+
+//nolint:paralleltest // We can't parallelize it because we have a limit number of real auth tokens.
+func TestService_GetUserByID_Success(t *testing.T) {
+	if testing.Short() {
+		return
+	}
+	ctx, cancel := context.WithTimeout(context.Background(), bridge.TestDeadline)
+	defer cancel()
+
+	userID, token := bridge.GetTestingAuthorizationAt(0)
+	var body string
+	var status int
+
+	SETUP("we cleanup existing users, if any", func() { bridge.MustDeleteAllUsers(ctx, t) })
+	GIVEN("create single user", func() {
+		bridge.MustCreateDefaultUser(ctx, t, userID, token, "bogus.user")
+	})
+	WHEN("get user by ID", func() {
+		body, status = bridge.GetUserByID(ctx, t, userID, token)
+	})
+	THEN(func() {
+		IT("is successfully returned", func() {
+			assert.Equal(t, 200, status)
+		})
+		IT("user received well", func() {
+			verifyUserProfile(t, body, userID, "bogus.user")
+		})
+	})
+}
+
+//nolint:paralleltest // We can't parallelize it because we have a limit number of real auth tokens.
+func TestService_GetUserByID_Failure_NoUser(t *testing.T) {
+	if testing.Short() {
+		return
+	}
+	ctx, cancel := context.WithTimeout(context.Background(), bridge.TestDeadline)
+	defer cancel()
+
+	userID, token := bridge.GetTestingAuthorizationAt(0)
+	var status int
+	var body string
+
+	SETUP("we cleanup existing users, if any", func() { bridge.MustDeleteAllUsers(ctx, t) })
+	GIVEN("empty database", func() {})
+	WHEN("get user by ID", func() {
+		body, status = bridge.GetUserByID(ctx, t, userID, token)
+	})
+	THEN(func() {
+		IT("http error 404", func() {
+			assert.Equal(t, 404, status)
+		})
+		IT("user not found", func() {
+			expected := `{"error":".+","code":"USER_NOT_FOUND"}`
+			bridge.AssertResponseBody(t, expected, body)
+		})
+	})
+}
+
+//nolint:paralleltest // We can't parallelize it because we have a limit number of real auth tokens.
+func TestService_GetUserByUsername_Success(t *testing.T) {
+	if testing.Short() {
+		return
+	}
+	ctx, cancel := context.WithTimeout(context.Background(), bridge.TestDeadline)
+	defer cancel()
+
+	userID, token := bridge.GetTestingAuthorizationAt(0)
+	var body string
+	var status int
+
+	SETUP("we cleanup existing users, if any", func() { bridge.MustDeleteAllUsers(ctx, t) })
+	GIVEN("create single user", func() {
+		bridge.MustCreateDefaultUser(ctx, t, userID, token, "bogus.user2")
+	})
+	WHEN("get user by username", func() {
+		body, status = bridge.GetUserByUsername(ctx, t, "bogus.user2", token)
+	})
+	THEN(func() {
+		IT("http ok", func() {
+			assert.Equal(t, 200, status)
+		})
+		IT("user received well", func() {
```

Comment:
```suggestion
		IT("received the user well", func() {
```
Because not _user received_, but _it received_ the user.

Reply:
the passive voice
user has been received well -> user received well
target destroyed
access granted
data acquired